app/uk/gov/hmrc/openidconnect/userinfo/config/AppContext.scala

```diff
@@ -27,4 +27,5 @@ object AppContext extends ServicesConfig {
   lazy val access = current.configuration.getConfig(s"$env.api.access")
   lazy val desEnvironment = current.configuration.getString(s"$env.microservice.services.des.environment").getOrElse(throw new RuntimeException(s"$env.microservice.services.des.environment is not configured"))
   lazy val desBearerToken =  current.configuration.getString(s"$env.microservice.services.des.bearer-token").getOrElse(throw new RuntimeException(s"$env.microservice.services.des.bearer-token is not configured"))
+  lazy val logUserInfoResponsePayload = current.configuration.underlying.getBoolean("log-user-info-response-payload")
```

Comment:
Does this getOrElse false

Reply:
it'll throw if the parameter can't be found - but because I've put it in `application.conf` this won't ever happen

Reply:
and the default value is `false`